src/Lykke.Service.Stellar.Api.Services/Horizon/HorizonService.cs

```diff
@@ -78,6 +89,15 @@ public async Task<List<TransactionDetails>> GetTransactions(string address, stri
             return new List<TransactionDetails>();
         }
 
+        public async Task<List<OperationResponse>> GetTransactionOperations(string hash)
+        {
+            var result = await new OperationsRequestBuilder(new Uri(_horizonUrl), _httpClientFactory.CreateClient())
```

Comment:
Just to be sure - is _httpClientFactory.CreateClient() memory leak safe?

Reply:
AFAIU, yes. See 
- https://docs.microsoft.com/ru-ru/aspnet/core/fundamentals/http-requests?view=aspnetcore-2.2#basic-usage
- https://docs.microsoft.com/en-us/dotnet/standard/microservices-architecture/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests#issues-with-the-original-httpclient-class-available-in-net-core